source/osh/datareceiver/DataSource.js

```diff
@@ -73,6 +73,44 @@ class DataSource {
         });
     }
 
+    /**
+     * Sets the startTime of the DataSource.
+     * @param {Number} startTime - the startTime (in millis)
+     */
+    setStartTime(startTime) {
```

Comment:
I think it may be better to set startTime/endTime/replaySpeed in a single method call and to disconnect/reconnect automatically

Reply:
So why don t use directly the updateUrl which allows to update any properties?

Reply:
Nothing guarantees that a datasource uses a URL so we should keep `updateUrl()` private. But I think it makes sense to expose time settings via a public API.

Reply:
we can replace setStart, setEnd and setReplaySpeed by setTime(startTime, endTime, replaySpeed), would be better?